src/clarity-angular/popover/dropdown/dropdown.ts

```diff
@@ -56,6 +61,11 @@ export class Dropdown {
     constructor(public elementRef: ElementRef, @SkipSelf() @Optional() public parent: Dropdown,
                 public ifOpenService: IfOpenService) {
         this.anchor = elementRef.nativeElement;
+
+        // Subscribe and broadcast openChanges from the service
+        ifOpenService.openChange.takeUntil(this.ngUnsubscribe).subscribe(change => {
```

Comment:
Yes, we decided to not go forward with the pattern because it'd require the additional import (dependency) and the gain seemed not significant. Not sure if this will change in the future though but for now, would it be ok if you reverted the pattern?